bin/alluxio

```diff
@@ -257,13 +257,8 @@ function main {
     runJavaClass "$@"
   ;;
   "runTest")
+    PARAMETER="configured" # instructs runTests to use the configured master address
```

Comment:
@calvinjia @apc999 Do you think it's ok to change the API for TestRunner as long as we keep the API for `bin/alluxio runTest(s)` the same?